assets/javascripts/app.js

```diff
@@ -1,3 +1,19 @@
 // assets/javascripts/app.js
 
-//= require_tree .
\ No newline at end of file
+//= require_tree .
+
+(function($) {
+
+  //Format <time> elements
+  $('.time-human-readable').each(function (){
```

Comment:
A pattern in the front end guide is to prefix classes used as js hooks with `js-`, so `js-time-human-readable`.